project-base/storefront/pages/articles/[articleSlug].tsx

```diff
@@ -36,6 +36,7 @@ const ArticleDetailPage: NextPage = () => {
             title={article?.seoTitle}
             description={article?.seoMetaDescription}
             breadcrumbs={article?.breadcrumb}
+            canonicalQueryParams={[]}
```

Comment:
I see this in multiple places. Can we maybe "get rid of it"? Maybe a default parameter?

Reply:
We have set of default parameters in canonical URL generator. I am passing empty array to override those defaults. I considered these options:

1. Passing another prop like `isWithoutCanonicalQueryParams` - no need for additional prop according to other options
2. Pass `null` to `canonicalQueryParams` to differentiate between `undefined` and `null` - use `null` to remove params and `undefined` to use the defaults, I didn't like to see `canonicalQueryParams !== null` which invokes me need to replace it with `!canonicalQueryParams` which then won't work
3. Pass whatever to queries be filtered with - the best option since it also reflect real array of override values

Reply:
Nice, thanks for the explanation, makes sense.